solvers/snapml.py

```diff
@@ -11,6 +13,13 @@ class Solver(BaseSolver):
     install_cmd = "conda"
     requirements = ["pip:snapml"]
 
+    parameters = {"gpu": [True, False]}
```

Comment:
Actually, the parameter per default of this solver would be the first one (implicitely, which is not great....)
So if you put this, the test should run without you needing to skip this:
```suggestion
    parameters = {"gpu": [False, True]}
```

Reply:
Thanks a lot! I'll use this for now but yes something more "intuitive" could be nicer here later